tests/Dockerfile.vic-integration-test

```diff
@@ -0,0 +1,15 @@
+# VIC Test Image
+#
+# Build a docker image that runs latest Ubuntu with golang, robotframework and vmware-vic installed
+# (https://golang.org/)
+# (http://robotframework.org/)
+# (https://github.com/vmware/vic)
+#
+# docker build -t vic-integration-test --no-cache -f Dockerfile.vic-integration-test .
+
+from vic-integration-test-base
+
+# Install latest version of vmware-vic
+run go get github.com/vmware/vic; echo 0
+run make -C /src/github.com/vmware/vic all
```

Comment:
Ah cool, I didn't realize - which docker file is that and where is it (I can't find it in the repo)? I can take a look at using that one.  The only one I know of is the golang:1.6.2, which I didn't think had VIC in it.

Mikkel Hagen
Quality Engineering Lead
Phone: (716) 307 0362
Address: 6500 River Pl Blvd, Austin, TX 78730

On 05/28/2016 07:35 PM, Faiyaz Ahmed wrote:

In tests/Dockerfile.vic-integration-test<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_vmware_vic_pull_707-23discussion-5Fr64996107&d=CwMCaQ&c=Sqcl0Ez6M0X8aeM67LKIiDJAXVeAw-YihVMNtXt-uEs&r=RWOEbpwLdzFOxVxJx1-lNmFIV0bt5SZaCb_OGY7aD4c&m=8_cWW01E8sjiCizxbL6uCRZwZ9-8PPig8eMdMR1t888&s=YnO0yLYnqk4pm-gg3Jrby5bYj0F71ahP3nNQTi_U9rs&e=>:

> @@ -0,0 +1,15 @@
> +# VIC Test Image
> +#
> +# Build a docker image that runs latest Ubuntu with golang, robotframework and vmware-vic installed
> +# (https://golang.org/)
> +# (http://robotframework.org/)
> +# (https://github.com/vmware/vic)
> +#
> +# docker build -t vic-integration-test --no-cache -f Dockerfile.vic-integration-test .
> +
> +from vic-integration-test-base
> +
> +# Install latest version of vmware-vic
> +run go get github.com/vmware/vic; echo 0
> +run make -C /src/github.com/vmware/vic all


So this dockerfile does nothing beyond building top of tree? We already have a Dockerfile for that. I'm not sure this is necessary. Can you explain why we need this?

-
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_vmware_vic_pull_707_files_d3c7fb8e6e6916ae2d1accbcaad7e9384f649be5-23r64996107&d=CwMCaQ&c=Sqcl0Ez6M0X8aeM67LKIiDJAXVeAw-YihVMNtXt-uEs&r=RWOEbpwLdzFOxVxJx1-lNmFIV0bt5SZaCb_OGY7aD4c&m=8_cWW01E8sjiCizxbL6uCRZwZ9-8PPig8eMdMR1t888&s=5Yjv22Qne1A79SWsHCAJYws1d6QAPU1NxGBx7PnEyZ8&e=>, or mute the thread<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe_ASfq-2DXfMRwnJWeUscaopP8zMA8qyeQBtks5qGN9CgaJpZM4Igvq6&d=CwMCaQ&c=Sqcl0Ez6M0X8aeM67LKIiDJAXVeAw-YihVMNtXt-uEs&r=RWOEbpwLdzFOxVxJx1-lNmFIV0bt5SZaCb_OGY7aD4c&m=8_cWW01E8sjiCizxbL6uCRZwZ9-8PPig8eMdMR1t888&s=ES8yTt9jcRTbqES1PoRSvNicEQsk1pTNTur5s6uBluc&e=>.